google/api_core/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "1.30.0"
+__version__ = "2.0.0-b1"
```

Comment:
Is this a new process, i.e. having a preliminary release instead of immediately switching to 2.0.0? Also, should this be beta or an RC?

FWIW, the approach itself does seem sensible to me.

Reply:
As a semver-major / breaking change, I thought it was reasonable to do a short beta.  I could be persuaded that it should be an RC, but I want folks to consider carefully whether they install it in production until we've gotten feedback that all is OK.